handbook/engineering/README.md

```diff
@@ -38,23 +38,14 @@ What happens next? The engineering output and architecture DRI reviews engineeri
 
 If there are product changes (i.e. interface, documentation, or dependency changes), the story is added to the "New requests" column on the drafting board.
 
-If there are no product changes, and the DRI decides to prioritize the story, the story is added to the "Specified" column on drafting board so that it can be estimated.
+If there are no product changes, and the DRI decides to prioritize the story, the story is added to the "Specified" column on the drafting board so that it can be estimated.
 
 > We prefer the term engineering-initiated stories over technical debt because the user story format helps keep us focused on our users and contributors.
 
-### Manage release branches
-Every three weeks, release a minor version of Fleet from the `main` branch. 
-
-Every week between minor releases, release a patch release with fixes for released bugs. 
-
-The DRI for publishing a release is responsible for creating the next patch branch. Create a patch branch off the latest tagged release of Fleet. Submit PRs for released bug fixes directly to the target patch branch to avoid merge conflicts later in the release cycle. After merging into the patch branch, submit another PR to `main` containing the same fix and resolve any merge conflicts.
-
-> It is the responsibility of the person merging the fix into the patch branch to make sure the fix is also merged into `main`.
-
 ### Fix a bug
-If the bug is labeled `~unreleased bug`, branch off and put your PR into `main`.
+If the bug is labeled `~unreleased bug`, branch off and put your PR into `main`. These issues can be closed as soon as they complete QA.
```

Comment:
Do you see a need to add: "The DRI for closing will typically be the QA engineer." ?

Reply:
I think that's implied but if there's confusion I will update.